packages/venia-ui/lib/components/Header/cartTrigger.js

```diff
@@ -26,26 +33,51 @@ const CartTrigger = props => {
     const buttonAriaLabel = `Toggle mini cart. You have ${itemCount} items in your cart.`;
     const itemCountDisplay = itemCount > 99 ? '99+' : itemCount;
 
-    const itemCounter = itemCount ? (
+    const maybeItemCounter = itemCount ? (
         <span className={classes.counter}>{itemCountDisplay}</span>
     ) : null;
+    const maybeMiniCartOpenIndicator = miniCartIsOpen ? (
+        <div className={classes.openIndicator} />
+    ) : null;
```

Comment:
We don't normally do presentational elements; ordinarily this would be a border or shadow on the button. Did you have a motive for this approach?

Reply:
Oops, didn't think it through - I'll change this

Reply:
Resolved by https://github.com/magento/pwa-studio/pull/2494/commits/a4cf8a0e41a32ca4655aa99ce6e8c165960bbe59.